### PR TITLE
Update tools

### PR DESCRIPTION
**Pull request by loftusa**
TODO:
- `lol` slide deck is old
- `ndmg` slide deck is old
- `graspy` slide deck is to the DiPy talk, and there might be a new one (@j1c ?)

### CODE REVIEW

**Review by falkben — CHANGES_REQUESTED** [review]
netlify failed -- maybe just needs a manual restart?

My understanding about this feature request was that we wanted to "feature" one slidedeck above the others, and that that one would go on the top bar.  Not that we'd be deleting existing slide decks.  @jovo can you confirm?

**Comment by loftusa** [comment]
I think we just want a link to a single slide deck:
```
this means that each such page:
- reg
- graspy
- etc

just has a link to a single slide deck.
ask me if you need to know which of several to choose.

neurodata.io/talks/mgc
neurodata.io/talks/rerf

are done

devin/dtward will make reg
not sure about others yet
```
Gonna check this locally, one sec

**Comment by loftusa** [comment]
@falkben should be good now, providing we are sticking with one slide

File: views/tool-page-base.html [hunk 1]
```diff
@@ -26,16 +32,6 @@ <h6>Publications</h6>
             </div>
         {% endif %}
 
-        {% if doc.slides %}
-        <div class="pubs">
-            <h6>Slides</h6>
-            <ol>
-            {% for item in doc.slides %}
-                <li><a target="_blank" href={{item.link}}>{{item.name}}</a></li>
```
Comment (falkben) [inline views/tool-page-base.html:34]: I don't think we actually want to delete this -- just replace w/ the single slide deck...

**Review by falkben — CHANGES_REQUESTED** [review]

**Review by falkben — CHANGES_REQUESTED** [review]
Can you please organize the yaml a little differently to be more obvious what "name" is used for.

I would do this:

```yaml
slide: 
  - name: "GrasPY is the worst"
  - link: "link.html"
```

then in the template actually use that name (currently you are not using the name at all).

also, i think it would look better without the giant link "slides" but instead in the same format as publications or at least the same font size

finally, @jovo wanted it in the navbar for each page, but since you renamed `slides` to `slide`, it won't render in the navbar

File: views/tool-page-base.html (hunk 1 above)
Comment (loftusa) [reply views/tool-page-base.html:34]: done

**Comment by loftusa** [comment]
@falkben This should be good now.

Only thing I didn't do is have slides in the same format as publications / seem font size, the rationale being because Jovo specifically said he wanted a slides link in the header.

Happy to change that as well if you have strong feelings about it.

**Review by falkben — CHANGES_REQUESTED** [review]
A link to the slides still missing on the top nav for each tool page. I believe there's a communication issue here.  That's what @jovo means by "in the header"

Having the name of the tool listed twice on each page does not look good to me.  I would prefer the same format as the publications because that will look better.  At minimum though, it should say slides: link to slides.  Think about the user when you are creating these things as that link wouldn't mean anything to anyone otherwise.  

I also think we can link multiple slide decks in that listing, but have the "single" slide deck be listed first, and be the one that is in the top nav.

**Comment by loftusa** [comment]
Oh. So, by top nav we mean the white bar on the top of each tools page?
Definitely misunderstood that, thanks.

Will do next monday ... sorry this particular task has dragged on so long!

**Comment by falkben** [comment]
> Oh. So, by top nav we mean the white bar on the top of each tools page?

correct.

**Review by loftusa — COMMENTED** [review]
changes addressed

**Review by loftusa — COMMENTED** [review]
(Some new updates to graspy pubs put into this PR as well)

**Review by falkben — APPROVED** [review]

**Pull request merged**